proposals/0306-actors.md

```diff
@@ -832,32 +787,70 @@ If we can address the above, task-chain reentrancy can be introduced into the ac
 
 ## Alternatives considered
 
-### Eliminating inheritance
+### Actor inheritance
+
+Earlier pitches and the first reviewed version of this proposal allowed actor inheritance. Actor inheritance followed the rules of class inheritance, albeit with specific additional rules required to maintain actor isolation:
+
+* An actor could not inherit from a class, and vice-versa.
+* An overriding declaration must not be more isolated than the overridden declaration.
 
-Like classes, actors as proposed allow inheritance. However, actors and classes cannot be co-mingled in an inheritance hierarchy, so there are essentially two different kinds of type hierarchies. It has been [proposed](https://docs.google.com/document/d/14e3p6yBt1kPrakLcEHV4C9mqNBkNibXIZsozdZ6E71c/edit#) that actors should not permit inheritance at all, because doing so would simplify actors: features such as method overriding, initializer inheritance, required and convenience initializers, and inheritance of protocol conformances would not need to be specified, and users would not need to consider them. The [discussion thread](https://forums.swift.org/t/actors-are-reference-types-but-why-classes/42281) on the proposal to eliminate inheritance provides several reasons to keep actor inheritance:
+Subsequent review discussion determined that the conceptual cost of actor inheritance outweighed its usefulness, so it has been removed from this proposal. The form that actor inheritance would take in the language is well-understand from prior iterations of this proposal and its implementation, so this feature could be re-introduced at a later time.
 
-* Actor inheritance makes it easier to port existing class hierarchies to get the benefits of actors. Without actor inheritance, such porting will also have to contend with (e.g.) replacing superclasses with protocols and explicitly-specified stored properties at the same time.
-* The lack of inheritance in actors won't prevent users from having to understand the complexities of inheritance, because inheritance will still be used pervasively with classes.
-* The design and implementation of actors naturally admits inheritance. Actors are fundamentally class-like, the semantics of inheritance follow directly from the need to maintain actor isolation. The implementation of actors is essentially as "special classes", so it supports all of these features out of the box. There is little benefit to the implementation from eliminating the possibility of inheritance of actors.
+### Cross-actor lets
```

Comment:
I'm confused what this section is claiming; 

Are we removing this "very permissiveness" from local actors or not by default or keeping it and this future work explains... I'm not sure what?
 
--- 

The permissive synchronous let access is definitely a bit "weird" and "not pure actor model" etc. but I also don't think it's a huge deal to do so. As long as the type allowed for such access is required to be `Sendable`. 

Distributed actors obviously will not allow such synchronous access either way, _even if_ `nonisolated let x: X` so I don't think it matters either way for distributed actors and the difference will exist anyway. Though one could reach into `distributed actor DA { nonisolated let x: X }` with `da.whenLocal { $0.x }` without having to do any extra "with unsafe actor state" dance etc. Long story short: distributed actors don't matter much for allowing cross-actor lets or not, because in all shapes (even non isolated) it's banned thing.

Reply:
We will call this out for discussion in the second review